app/packages/core/src/components/ViewBar/ViewStage/viewStageParameterMachine.ts

```diff
@@ -152,7 +152,7 @@ export const PARSER = {
       } catch {
         array = stripped.split(",");
       }
-      return Array.isArray(array) && array.every((e) => PARSER.str.validate(e));
+      return Array.isArray(value) && array.every((e) => PARSER.str.validate(e));
```

Comment:
Lanny, not sure we need to change these. the change causing an issue where the value disappears when I use MatchTags() 🤔 

my 2cents as to why we don't need these changes
User can pass an object to the field that could potentially be parsed to an array ex: `'["dd", "cc"]';` -> ["dd", "cc"] // type is array

User can also pass an object to the field that could be parsed into an object (hence not an array). which the validation should fail and it would in `Array.isArray(array)` part

what do you think?

Reply:
Previously when value is a string ("kite in the sky"), it can pass the `Array.isArray(array)` check and will then gets treated like a str<list>, then cause the build to break, as it doesn't have the array method. 

I see what you are pointing out. What I did on "similarity-textprompt" branch earlier was to add an additional condition to make sure value is not string type. I will revert to that.